tests/functional/test_libraries.py

```diff
@@ -489,3 +489,37 @@ def test_pandas_extension(pyi_builder):
         """)
 
 
```

Comment:
Please put the test alphabetically into the file. THis makes it easer to find them.